Desktop/analysis/options/optionterms.h

```diff
@@ -27,7 +27,6 @@ class OptionTerms : public OptionI<std::vector<std::vector<std::string> > >
 public:
 			OptionTerms() : OptionI(true), _onlyOneComponent(false), _onlyOneTerm(false) 
 			{
-				setShouldEncode(true); //Maybe we should check if the terms are in fact columnnames or not? But, for now I will just assume that terms are very likely to contain columnnames. And otherwise Ill have to override asMetaJson and step through all this like in OptionTerms::asJSON() and simply check it
```

Comment:
Did you take the meta into account after removing this?

Reply:
OptionTerms is never used directly, only as subclass of OptionVariables (which is always encoded), or of OptionTerm. OptionTerm is used in CustomContrast table, JAGS and ComponentList for values that are not variables, and in BoundControlTerms where I have added code so that it sets the encoding only when the model contains variables.
I hope to remove all these option classes soon...

Reply:
https://github.com/jasp-stats/INTERNAL-jasp/issues/1163

Reply:
Yes the JASP file in this issue still works.